scripts/cms-any/dispatch-ci.php

```diff
@@ -23,13 +23,13 @@
 
 jobs:
   dispatch-ci:
-  name: Dispatch CI
-  # Only run cron on the $account account
-  if: (github.event_name == 'schedule' && github.repository_owner == '$account') || (github.event_name != 'schedule')
-  runs-on: ubuntu-latest
-  steps:
-    - name: Dispatch CI
-      uses: silverstripe/gha-dispatch-ci@v1
+    name: Dispatch CI
+    # Only run cron on the $account account
+    if: (github.event_name == 'schedule' && github.repository_owner == '$account') || (github.event_name != 'schedule')
+    runs-on: ubuntu-latest
+    steps:
+        - name: Dispatch CI
```

Comment:
```suggestion
      - name: Dispatch CI
```
Over-indented

Reply:
Also PR prefix should be MNT

Reply:
Curse my IDE! That's probably how it happened in the first place 😅
Fixed.